schema/fleet_schema.json

```
@@ -1,7 +1,7 @@
 [
   {
     "name": "account_policy_data",
-    "examples": "Query the creation date of user accounts. You could also query the date of the last failed login attempt or password change.\n```\nSELECT strftime('%Y-%m-%d %H:%M:%S',creation_time,'unixepoch') AS creationdate FROM account_policy_data;\n```"
+    "examples": "Query the creation date of user accounts. You could also query the date of the last failed login attempt or password change.\n```\nSELECT strftime('%Y-%m-%d %H:%M:%S',creation_time,'unixepoch') AS creationdate FROM account_policy_data\n```\n\nSee each user's last password set date and number of failed logins since last successful login to detect any intrusion attempts.\n\n```SELECT \n    u.username\n    u.uid, \n    strftime('%Y-%m-%dT%H:%M:%S', a.password_last_set_time, 'unixepoch') AS password_last_set_time,\n    a.failed_login_count,\n    strftime('%Y-%m-%dT%H:%M:%S', a.failed_login_timestamp, 'unixepoch') AS failed_login_timestamp\nFROM account_policy_data AS a\nCROSS JOIN users AS u USING (uid)\nORDER BY password_last_set_time ASC\n;\n```"
```

```suggestion
    "examples": "Query the creation date of user accounts. You could also query the date of the last failed login attempt or password change.\n```\nSELECT strftime('%Y-%m-%d %H:%M:%S',creation_time,'unixepoch') AS creationdate FROM account_policy_data;\n```\n\nSee each user's last password set date and number of failed logins since last successful login to detect any intrusion attempts.\n\n```SELECT \n    u.username\n    u.uid, \n    strftime('%Y-%m-%dT%H:%M:%S', a.password_last_set_time, 'unixepoch') AS password_last_set_time,\n    a.failed_login_count,\n    strftime('%Y-%m-%dT%H:%M:%S', a.failed_login_timestamp, 'unixepoch') AS failed_login_timestamp\nFROM account_policy_data AS a\nCROSS JOIN users AS u USING (uid)\nORDER BY password_last_set_time ASC\n;\n```"
```

Stop removing my ';`s 😂

---

:( Sorry... hard to parse this string... We need to improve the UX of updating this!

---

It would be nicer if each example was separate, and then if the description and query had their own fields as well, that is for sure!